homeassistant/components/life360/strings.json

```
@@ -14,7 +14,7 @@
       "invalid_username": "Invalid username",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "user_already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
-      "unexpected": "Unexpected error communicating with Life360 server"
+      "unexpected": "[%key:common::config_flow::error::unknown%]"
```

We prefer to also use the key `unknown` for this error.
```suggestion
      "unknown": "[%key:common::config_flow::error::unknown%]"
```